cli/src/cli.ts

```diff
@@ -30,12 +29,12 @@ async function runUser (task: Task, server: Server, data: data.DataSplit): Promi
 }
 
 async function main (): Promise<void> {
-  const server = await startServer()
+  const [server, url] = await startServer()
```

Comment:
What about making the url a server attribute in the future?

Reply:
there is two type of server exposed currently, `node:http.Server` & our `Disco` server. wrapping node's one fully inside `Disco` would be better, and maybe adding url attribute can be useful.
note that this field would `URL | undefined`, as the server has two states, started or not. using a function to start the server which return an URL ensures that the only way to get the value is by starting it. I find it quite elegant rather than to check `server.url !== undefined` to know that.

Reply:
Indeed makes sense!